mmv1/products/cloudrun/terraform.yaml

```diff
@@ -390,6 +398,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       spec.template.spec.containers.resources.limits: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+      spec.template.spec.containers.startupProbe: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      spec.template.spec.containers.startupProbe.tcpSocket.port: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      spec.template.spec.containers.startupProbe.httpGet.path: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
This indicates that the server will pick a non-fixed default for path when not provided. Is that correct?

Reply:
The rules are:

- Not specified -> "/"
- Empty string -> "/"
- Other string -> keep the input

Reply:
In that case, should we consider a client-side `default_value` or `"/"`? That means that if a user doesn't specify anything, they'll get `"/"` which is the same behaviour as the API.

`default_from_api` preserves values when the field is unset rather than resets to default, so it's generally undesirable if the default value is fixed.

Reply:
I tried `default_value = "/"`.

If I don't specify `path`, the test passed.

If I specify `path=""`, the test failed due to

```
~ http_get {
        - path = "/" -> null

          # (3 unchanged blocks hidden)
      }
```

I think Terraform treats "" as an input, not unset.